### PR TITLE
EZP-27780: Remove unused table from legacy SearchIndex

### PR DESCRIPTION
**Pull request by andrerom**
Updated upgrade doc for 6.11 for fixing schema.

### CODE REVIEW

**Comment by DominikaK** [comment]
@andrerom: I reformulated the text for clarity. Could you take a look if the meaning has not been distorted?

File: docs/releases/updating_ez_platform.md [hunk 1]
```diff
@@ -133,13 +133,13 @@ in `app/AppKernel.php`.
...
 
-This step is only relevant for some releases:
+These steps are only relevant for some releases:
 
 **ezsearch_return_count table removal**
 
 V1.11
 
-v1.11.0 removes `ezsearch_return_count` table, aligning with eZ Publish legacy since 5.4/2014.11 to avoid issues for those using upgrade install with our withouth legacy bridge. To fix this apply the following database update script if your install is not an eZ Publish upgrade where it was already removed:
+v1.11.0 removes the `ezsearch_return_count` table, which had been obsolete in eZ Publish legacy since 5.4/2014.11. This avoids issues which would occur when you upgrade using legacy bridge. Apply the following database update script if your installation has not had the table removed by an earlier eZ Publish upgrade:
```
Comment (andrerom) [inline docs/releases/updating_ez_platform.md]: table was removed, not obsoleted. so wording there is a bit wrong. otherwise it looks ok

**Comment by andrerom** [comment]
Note, the update SQL is missing form the release: https://github.com/ezsystems/ezpublish-kernel/compare/v6.11.0...6.11

So will need to resolve that with @bdunogier before we can merge this.

**Comment by andrerom** [comment]
Issue above is resolved, the final build contains the files needed for this.

@DominikaK you can go ahead with this ;)

**Comment by mere-teresa** [comment]
To be sure, could you confirm the _affected versions_ are those on the JIRA Issue, @andrerom?

>  dev-master, 1.7.4, 1.10.1

**Comment by andrerom** [comment]
Yes, but this is upgrade doc telling people to upgrade, so it can only happen when upgrading to 1.11. So this is therefor pointing towards 1.11 branch.

**Pull request merged**